README.md

```diff
@@ -1,5 +1,39 @@
-## Restart Repair Directory
+=== Repair Directory ===
+Contributors: The Restart Project
+Donate link: https://therestartproject.org/give/
+Tags: repair
+Requires at least: 5.6.1
+Tested up to: 5.6.1
+Stable tag: master
+Requires PHP: 7.2
+License: GPLv3 or later
+License URI: http://www.gnu.org/licenses/gpl-3.0.html
+
+The Restart Repair Directory is a tool for encouraging repair through local businesses.  
+
+== Description ==
 
 The Restart Repair Directory is a tool for encouraging repair through local businesses.  
 It allows for the recording of local repair businesses in a directory and the searching of this directory by 
-the public.  [Find out more](https://therestartproject.org/repairdirectory/about).
\ No newline at end of file
+the public.  [Find out more](https://therestartproject.org/repairdirectory/about).
+
+== Installation ==
+
+This plugin is manually installed from a ZIP file supplied by The Restart Project.
+
+1. Go to the WordPress Plugins Page
+2. Click *Add New*
+3. Select the ZIP file
+4. Activate the plugin
+
+Once installed and activated, the plugin can be used on a WordPress page using the `repair-directory` shortcode, e.g.
+
+`[repair-directory region="Wales" width="100%" height="500" region="London" domain="https://map.restarters.net"`
+
+The following options are available:
+* **domain** default `https://map.restarters.net`.  This is the URL of your domain, which is including the Repair 
```

Comment:
Presume this should be the full URL of the embedding page, not just domain, e.g. https://therestartproject.org/repairdirectory ?

Reply:
Yes.  Pushed a change to clarify.

Reply:
Thanks - actually I was thinking about the path.  Given that the parameter is called domain, people might then think this means just the domain.  But presumably this parameter also allows the inclusion of a path?

So for example on trp.org the parameter would be e.g.:

```
domain="https://therestartproject.org/repairdirectory
```

I wonder if the parameter name should be changed to 'url'.